real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/simulation/SimulatedBuggy.java

```diff
@@ -40,9 +40,9 @@ public void setBrakesDown(boolean brakesDown) {
 	private double  y = 0.0;
 	private double  th = 0.0;
 	private double wheelTh = 0.0;//the orintation in buggy coordinates of the front wheel 
+	private double desiredWheelTh = 0.0;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2446228622/issues/source?bid=2938665&fileBranchId=3246677#l43)